### DISCUSSION TITLE
What are you using to phonemize text for piper models

### DISCUSSION THREAD

**Discussion by danemadsen** [k2-fsa/sherpa-onnx#686] [Q&A] [answered]
Hi, I'm wanting to create a simple implementation of piper models in my own project to be integrated into dart. The official piper cpp library uses e-speak to phonemize but you don't seem to be using espeak. If you're using your own code to phonemize could you please point me to the area where that's being done?

Thank you in advance

**Comment by csukuangfj** [comment, marked as answer]
We are indeed using espeak-ng, like exactly what piper is doing.

Please see #379

There is an on-going work for flutter.

**Reply by danemadsen** [reply]
Thanks for the info
